packages/core/src/components/button/_common.scss

```diff
@@ -393,6 +393,7 @@ $button-intents: (
 
     &:disabled,
     &.pt-disabled {
+      background: none;
```

Comment:
Yeah, we used to do that in some places. The advantage is that it's less lines of code, but it's harder to maintain. I prefer keeping it super explicit, even if it adds a few more lines